jena-arq/src/main/java/org/apache/jena/sparql/function/scripting/NV.java

```diff
@@ -29,9 +29,14 @@
 import org.apache.jena.sparql.lib.RDFTerm2Json;
 
 /**
- * General representation of an {@link NodeValue} for JavaScript. Conversion is to native
- * types where possible, otherwise {@code NV}. {@code NV.toString} of a URI returns the
- * uri as a string so {@code NV} works naturally in Java/JavaScript.
+ * General representation of an {@link NodeValue} for JavaScript and other scripting
+ * languages used to write SPARQL custom functions.
+ * Class {@link NV} provides access to all the details of a {@link NodeValue} for cases where
+ * translation to native types isn't enough.
+ *
+ * Conversion is to native types where
```

Comment:
`<p>` needed for separate paragraph in javadocs? (not sure if still needed, haven't read latest changes in jvm releases)